src/main/java/VisualLogic/variables/VS1DInteger.java

```diff
@@ -74,30 +77,32 @@ public void copyValueFrom(Object in) {
     }
 
     public void loadFromStream(java.io.FileInputStream fis) {
-        try {
-            DataInputStream dis = new DataInputStream(fis);
+        try (DataInputStream dis = new DataInputStream(fis)) {
+
 
             int size = dis.readInt();
             value = new int[size];
 
             for (int i = 0; i < value.length; i++) {
                 value[i] = dis.readInt();
             }
-        } catch (Exception ex) {
+        } catch (IOException ex) {
+            Logger.error(ex);
             System.err.println("Fehler in VS1DInteger.loadFromStream() : " + ex.toString());
         }
     }
 
     public void saveToStream(java.io.FileOutputStream fos) {
-        try {
-            DataOutputStream dos = new DataOutputStream(fos);
+        try (DataOutputStream dos = new DataOutputStream(fos)) {
+
             dos.writeInt(value.length);
 
             for (int i = 0; i < value.length; i++) {
                 int val = value[i];
                 dos.writeInt(val);
             }
-        } catch (Exception ex) {
+        } catch (IOException ex) {
+            Logger.error(ex);
             System.err.println("Fehler in VS1DInteger.saveToStream() : " + ex.toString());
```

Comment:
`System.err` call should be removed.

Reply:
Should we fix this in this PR too?

Reply:
Depending on how many occurrences we are speaking of this may be a separate PR.

Reply:
```bash
❯ rg --count "System.err.println" src/ | cut -d':' -f 2 | count
16
```

Reply:
I thinks that is manageable to include in here.

Reply:
alright